dogs.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt a Dog</title>
+    <link rel="stylesheet" type="text/css" href="styles/stylesheet.css">
+  </head>
+
+  <body>
+
+    <header style="background-color:#ADD8E6;">
+      <img src="images/adoptadog-logo.svg" alt="white adopt a dog logo"  width="503" height="136">
+      <h1>Seattle's premiere dog adoption agency</h1>
+      <img src="images/cart-icon.svg" alt="shopping cart icon" width="29" height="26">
+      <p>$1234.56</p>
+      <a class="checkout-button" href="checkout.html">checkout</a>
+    </header>
+
+    <nav style="background-color:#C0C0C0">
+      <a href="index.html">Home</a> /
+      <a href="dogs.html">Dogs</a> /
+      <a href="blog.html">Blog</a>
+    </nav>
+
+    <section class="gallery">
+      <h2>Here are all of our Dogs!</h2>
+      <figure class="gallery-item">
+        <img src="images/murphy-card.jpg" alt="a picture of a brown dogs face" width="250">
+        <figcaption>
+          <h3>Murphy</h3>
+          <p>Cost to Adopt: $123.45</p>
+          <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+          <a class="adopt-button" href="checkout.html">Adopt</a>
```

Comment:
You can make these buttons into buttons. Don't worry about them linking anywhere, they'll do stuff later on in the course.